Source/WebCore/page/ChromeClient.h

```diff
@@ -394,6 +394,7 @@ class ChromeClient {
     virtual void triggerRenderingUpdate() = 0;
     // Schedule a rendering update that coordinates with display refresh. Returns true if scheduled. (This is only used by SVGImageChromeClient.)
     virtual bool scheduleRenderingUpdate() { return false; }
+    virtual void renderingUpdateFrequencyChanged() { }
```

Comment:
Maybe use the "RenderingUpdateFramesPerSecond" terminology that we use elsewhere.